packages/alchemy/src/gas-manager.ts

```diff
@@ -4,18 +4,15 @@ import {
   arbitrumSepolia,
   base,
   baseSepolia,
-  fraxtal,
-  fraxtalTestnet,
   mainnet,
+  sepolia,
   optimism,
   optimismSepolia,
   polygon,
   polygonAmoy,
   polygonMumbai,
-  sepolia,
-  zora,
-  zoraSepolia,
-} from "viem/chains";
+} from "@alchemy/aa-core";
```

Comment:
Was there a reason we were importing from `viem` instead of `aa-core` before?

Reply:
nope I think that was a mistake, good catch